src/data/fin.lean

```diff
@@ -1012,6 +1012,168 @@ def append {α : Type*} {o : ℕ} (ho : o = m + n) (u : fin m → α) (v : fin n
   then u ⟨i, h⟩
   else v ⟨(i : ℕ) - m, (nat.sub_lt_left_iff_lt_add (le_of_not_lt h)).2 (ho ▸ i.property)⟩
 
+variables {ι : Type*} {o : ℕ} {u : fin m → ι} {v : fin n → ι}
```

Comment:
`ι` seems like an odd choice of name, since usually it is used as `ι → ...` - `β` would be less surprising.